src/pages/Organize/Folder/Create/index.js

```diff
@@ -14,12 +14,139 @@
  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-***************************************************************/
+*/
 
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Spinner, Alert } from "react-bootstrap";
+import InputContainer from "../../../../components/Widgets/Input";
+import Button from "../../../../components/Widgets/Button";
+import { getAllFolders, createFolder } from "../../../../services/folders";
 
 const CreateFolder = () => {
-  return <div>Create Folder</div>;
+  const initialState = {
+    parentFolder: 1,
+    folderName: "",
+    folderDescription: "",
+  };
+  const initialFolderList = [
+    {
+      id: 1,
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  const initialMessage = {
+    type: "success",
+    text: "",
+  };
+  const [createFolderData, setCreateFolderData] = useState(initialState);
+  const [folderList, setFolderList] = useState(initialFolderList);
+  const [loading, setLoading] = useState(false);
+  const [showMessage, setShowMessage] = useState(false);
+  const [message, setMessage] = useState(initialMessage);
+  const { parentFolder, folderName, folderDescription } = createFolderData;
+
+  const handleChange = (name) => (event) => {
+    setCreateFolderData({ ...createFolderData, [name]: event.target.value });
+  };
```

Comment:
@Aman-Codes The better way we can do is just pas the event variable as parameter and use`[e.target.name]`